justfile

```diff
@@ -0,0 +1,62 @@
+name := 'cosmic-osd'
+
+polkit-agent-helper-1 := /usr/libexec/polkit-agent-helper-1
+
+rootdir := ''
+prefix := '/usr'
+
+base-dir := absolute_path(clean(rootdir / prefix))
+
+bin-src := 'target' / 'release' / name
```

Comment:
Suggestion: Change this line to `bin-src := cargo-target-dir / 'release' / name` to use the previously read environment variable instead of hardcoding the target directory.